### ISSUE TITLE
unexpected taskstats structure size

### ISSUE THREAD

**Issue by lijinglin2019** [mdlayher/taskstats#9]
Hi, I encountered some errors while using this package. Below are the code and errors.

```go
$ cat main.go 
package main

import (
	"fmt"
	"github.com/mdlayher/taskstats"
	"os"
)

func main() {
	c, err := taskstats.New()
	if err != nil {
		panic(err)
	}
	stat, err := c.PID(os.Getpid())
	if err != nil {
		panic(err)
	}
	fmt.Printf("%+v", stat)
}
```

```bash
$ sudo -E go run main.go
panic: unexpected taskstats structure size, want 344, got 328

goroutine 1 [running]:
main.main()
	/home/lijinglin/go/src/github.com/lijinglin2019/golang/main.go:16 +0xec
exit status 2
```

**Comment by lijinglin2019** [comment]
The OS I am using
```bash
$ screenfetch 

 ██████████████████  ████████     lijinglin@thinkpad
 ██████████████████  ████████     OS: Manjaro 18.0.2 Illyria
 ██████████████████  ████████     Kernel: x86_64 Linux 4.19.13-1-MANJARO
 ██████████████████  ████████     Uptime: 3h 44m
 ████████            ████████     Packages: 1193
 ████████  ████████  ████████     Shell: zsh 5.6.2
 ████████  ████████  ████████     Resolution: 1920x1080
 ████████  ████████  ████████     DE: GNOME 
 ████████  ████████  ████████     WM: GNOME Shell
 ████████  ████████  ████████     WM Theme: Adapta-Nokto-Eta-Maia
 ████████  ████████  ████████     GTK Theme: Adapta-Nokto-Eta-Maia [GTK2/3]
 ████████  ████████  ████████     Icon Theme: Papirus-Adapta-Nokto-Maia
 ████████  ████████  ████████     Font: Noto Sans 11
 ████████  ████████  ████████     CPU: Intel Core i5-7200U @ 4x 3.1GHz [50.0°C]
                                  GPU: Mesa DRI Intel(R) HD Graphics 620 (Kaby Lake GT2) 
                                  RAM: 2723MiB / 7861MiB

```

**Comment by mdlayher** [comment]
I'm busy for a couple of days, apologies!  Perhaps @tklauser knows?

**Comment by tklauser** [comment]
@lijinglin2019 Are you working on the latest master? I think this issue should be fixed by #8 which takes care of the updated taskstats size for taskstats v9.

**Comment by lijinglin2019** [comment]
@tklauser https://github.com/mdlayher/taskstats/pull/8 works! 
Thank you very much!

> @lijinglin2019 Are you working on the latest master? I think this issue should be fixed by #8 which takes care of the updated taskstats size for taskstats v9.
